tools/nav2_bashrc

```diff
@@ -0,0 +1,14 @@
+OVERLAY_DIR=/opt/overlay_ws
+
+# https://docs.ros.org/en/rolling/Tutorials/Workspace/Creating-A-Workspace.html#source-the-overlay
+OVERLAY=$OVERLAY_DIR/install/setup.bash
+UNDERLAY=/opt/ros/${ROS_DISTRO}/setup.bash
+if [ -f "$OVERLAY" ]; then
```

Comment:
Don't think we should add this kind of logic to a one off script. Let's add an alias to the .bashrc instead, as we still may need to open a shell without automatically sourcing a workspace.

Reply:
> as we still may need to open a shell without automatically sourcing a workspace

I'm curious to know which use cases would be affected by the auto-sourcing of the workspace? 

Adding an alias to the bashrc doesn't add much more value to manually doing it (once it's done once and in the bash history, it's easy to pull up).